ccx_messaging/publishers/kafka_publisher.py

```diff
@@ -41,6 +42,7 @@ def __init__(self, outgoing_topic: str, kafka_broker_config: dict = None, **kwar
             raise CCXMessagingError("outgoing_topic should be a str")
 
         self.topic = outgoing_topic
+        self.compression = kwargs.get("compression")
```

Comment:
shall we parse this to a bool somehow? imagine the user configures the YAML value as "false". If self.compression treats it as a string, `if self.compression:` will be triggered

Reply:
because it is able to convert "" to false nil to false "true" to true etc

Reply:
You can put a default to False, something like

`bool(kwargs.get("compression", False))`